scripts/partition_manager/partition_manager.rst

```diff
@@ -8,9 +8,8 @@ When creating an application that requires child images (for example, a bootload
 
 See :ref:`ug_multi_image` and *Building and Configuring multiple images* in :ref:`zephyr:application` for more information about multi-image builds.
 
-By default, all |NCS| samples that require multiple images are configured to use the Partition Manager.
-If you prefer to build and program each image separately and disable the Partition Manager, configure your parent image (the application) to skip building the child images.
-Note that the Partition Manager is invoked only in multi-image builds.
+Partition Manager will be activated when there are multiple images
```

Comment:
Change to:
The Partition Manager is activated for all multi-image builds, no matter what build strategy is used for the child image.

(Also, keep the sentence on one line.)

Reply:
Done